GA/post.py

```
@@ -381,22 +399,17 @@ def _plot_blading_bar_chart(self, stage_idx, reference_blading, optimised_bladin
         x = np.arange(len(keys))
         base_bar_width = 0.8 / (num_indiv + 1)
 
-        colormap = list(colormaps["tab20"].colors)
-
-        indiv_colors = [colormap[i % len(colormap)] for i in range(num_indiv)]
-
-        plt.figure(f"Bar Chart with blading parameters for stage {stage_idx}")
-
+        plt.figure(f"Bar Chart with blading parameters", figsize=get_figsize(wf=1))
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)